tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -1349,4 +1349,86 @@ public function provide_test_build_line_items(): array {
+				'button_locations' => [ 'product', 'cart' ],
+				'location'         => 'cart',
+				'expected'         => true,
+			],
+			'ECE enabled, cart not in locations, checking cart' => [
+				'express_checkout' => 'yes',
+				'button_locations' => [ 'product' ],
+				'location'         => 'cart',
+				'expected'         => false,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_test_should_load_scripts_for_ece_location is not named in camelCase. (undefined)

(CamelCaseMethodName)